.github/workflows/build.yml

```
@@ -23,7 +45,7 @@ jobs:
         run: pip install . -v --no-deps
         env:
           NVTE_FRAMEWORK: pytorch
-          MAX_JOBS: 1
+          MAX_JOBS: 4
```

The PyTorch build fails even with `MAX_JOBS=4`:
```suggestion
          MAX_JOBS: 1
```